feat: delta action model related

FMint opened the pull request:
1. change the ppo class in use
2. rewrite the step method
3. add the command to select step

FMint committed:
feat: delta action model related

1. change the ppo class in use
2. rewrite the step method
3. add the command to select step

FMint committed:
feat: debug

kvkcon commented on README.md:
```diff
@@ -143,6 +143,34 @@ This is the visualization of the policy after traning 5800 iters. The policy is
++simulator=isaacgym \
++exp=motion_tracking \
++domain_rand=NO_domain_rand \
++rewards=motion_tracking/reward_motion_tracking_basic \
```
why did you choose this reward config

FMint replied:
delta_training? need modified?

kvkcon replied:
are you currently using `motion_tracking/reward_motion_tracking_basic` in delta training exps

kvkcon commented on humanoidverse/config/env/motion_tracking.yaml:
```diff
@@ -53,6 +53,7 @@ env:
     use_teleop_control: False
     enforce_randomize_motion_start_eval: False
 
+    use_delta_policy: False
```
maybe
`use_delta_policy: True`

FMint replied:
when training delta, the command line is explicitly defined `use_delta_policy=true.`
i think this is conducive to the unification of different training stages.

FMint committed:
feat: debug delta train

1. ppo.yaml add policy_checkpoint
2. take obs_dict['actor_obs'] as the input of policy1

FMint committed:
feat: debug

filipul1s commented:
May I ask how your reward scale is designed? The lower body of the dance movements I trained has difficulty following the movements, especially after adding dr, the model becomes less convergent. What dr must be added?
